src/screens/OrganizationEvents/OrganizationEvents.tsx

```diff
@@ -138,75 +137,19 @@ function OrganizationEvents(): JSX.Element {
     window.location.assign('/orglist');
   }
 
-  /* istanbul ignore next */
-
-  const handleSearchByTitle = (e: any) => {
-    const { value } = e.target;
-    const filterData = {
-      organization_id: currentUrl,
-      title_contains: value,
-    };
-    refetch(filterData);
-  };
```

Comment:
As you can see here, the filtering indeed does work (via an API call to the backend).